MathLib/Point3d.h

```diff
@@ -16,7 +16,12 @@
 
 #include <limits>
 
+#ifdef OGS_USE_EIGEN
+#include <Eigen/Eigen>
+#endif
```

Comment:
I think Eigen is not needed here. I forgot to remove these lines. @norihiro-w : can you remove the lines and merge the changes to the corresponding commit. By the way, I think the commits  c7deda4 and  ee35769 can also be squashed together. If you want do this go ahead.